### ISSUE TITLE
Image don't save!

### ISSUE THREAD

**Issue by wanderkpot** [filfat/swiftsnapper#105]
While I was using swiftsnapper I noticed that if I take some pictures, it doesn't save in my phone!
I'm using the last W10 build and lumia 830.
Sorry for bad english, I'm not a native speaker!

**Comment by craycraycoco** [comment]
I don't think it is supposed to save. I think it's more of a debug feature to make sure it can actually snap a photo. Correct me if I am wrong @filfat

**Comment by Ticomware** [comment]
@craycraycoco This is an expected behavior from a Snapchat user. In Snapchat, when a user takes a photo, the photo is saved to the device. SwiftSnapper doesn't yet as far as I know.

**Comment by ericbsantana** [comment]
When you take a photo on Snapchat, as far as I know, you can choose if you want to save it or not.

**Comment by Ahuge** [comment]
So far the photo's aren't being saved.
